packages/waku/src/lib/plugins/vite-plugin-rsc-index.ts

```diff
@@ -79,10 +77,13 @@ ${opts.htmlHead}
     },
     transformIndexHtml() {
       return [
+        // HACK: vite won't inject __vite__injectQuery anymore
```

Comment:
@Aslemammad What does it mean???

Reply:
Vite optimizes `import()` so it adds `?import` to imported urls. That'd cause double module hazard! 

This way, I hack it to use a global function so it does not get optimized.